### PR TITLE
Updates Kilo Station and re-adds it to rotation

### PR DESCRIPTION
**Pull request by dog132-ss13**
## About The Pull Request

Kilo Station has been updated to the current Beestation experience - namely adding an exploration dock as well as random maintenance.

## Why It's Good For The Game

Everyone loves Kilo. The return of this classic map has been long anticipated.

## Changelog
:cl:dog132
config: Kilo Station is part of the map rotation
add: Kilo Station comes with an exploration shuttle
add: Kilo Station comes with random maint
add: 4 random maint rooms from Kilo Station
del: Removed old maint rooms from Kilo Station
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
After 9 years in development, hopefully, it would've been worth the wait.

**Comment by ghost** [comment]
holy shit they really just can't let kilo die in peace.

**Comment by ItsTisme** [comment]
b-b-b-b-ba-bab-bb-b-ba-bas-ba-ba-BASED?!?!??!

**Comment by williamcll** [comment]
> holy shit they really just can't let kilo die in peace.

kilo is pog, just bring back all the maps.

**Comment by KubeRoot** [comment]
I think mapdiffbot shat itself because I can't see exploration on the images it produced. Is a screenshot of the full map using the map editor small enough to upload to github?

**Comment by dog132-ss13** [comment]
kilo_full
You'll find the Exploration Dock just east of Xenobiology.

**Comment by DatBoiTim** [comment]
Roidstation when?

**Comment by Archanial** [comment]
I will testmerge it when we finally merge fland.

**Comment by dog132-ss13** [comment]
Awesome, please remember to ping me on discord when you do.

**Comment by heepox** [comment]
nice... this would be good during the low pop hours

**Comment by Sweettoothart** [comment]
Could stasis beds be added to medical? Im not seeing any

File: config/maps.txt [hunk 1]
```diff
@@ -41,12 +41,11 @@ map deltastation
 	votable
 endmap
 
-#Does not support fastmos
+#Does not support fastmos (who cares????)
```
Comment (PowerfulBacon) [inline config/maps.txt]: remove the comment

**Comment by ike709** [comment]
🔑

**Comment by imsynthex** [comment]
> 🔑

can you at least explain it? is there something wrong with the PR or do you just have a burning hatred for kilo?

**Comment by Howluinb** [comment]
hype for kilo tbh please don't kill it off ike

**Comment by dog132-ss13** [comment]
A new hallway was added to connect Science and Exploration.
image

**Comment by dog132-ss13** [comment]
now the hallway looks good
image

**Comment by dog132-ss13** [comment]
image

**Review by Archanial — DISMISSED** [review]

**Review by Archanial — APPROVED** [review]

**Pull request merged**